frontend/src/components/SCDataGrid.tsx

```diff
@@ -33,7 +33,7 @@ export default function SCDataGrid({
   tank
 }: {
   employeeId: number | undefined;
-  tank: UpdateTankMetaData | undefined;
+  tank: tankSchema | undefined;
```

Comment:
purely to round out tthe below comments, I like TankMetaData, if that's what this model represents, better than TankSchema. The former tells you what type of data is represented by the model, the latter gives some ambiguous typing relation and that it is coming from the database.